api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/ChildDateOfBirthCalculator.java

```diff
@@ -29,30 +31,26 @@ public ChildDateOfBirthCalculator(PaymentCycleEntitlementCalculator paymentCycle
      * of the next cycle) are included.
      *
      * @param paymentCycle The current PaymentCycle including the relevant dates of birth of the children.
-     * @return The number of children who will soon turn 4 that affect the next Payment.
+     * @return The number of children who will soon turn 1 or 4 that affect the next Payment.
      */
-    public int getNumberOfChildrenTurningFourAffectingNextPayment(PaymentCycle paymentCycle) {
-        return getNumberOfChildrenWithBirthdayAffectingNextPayment(paymentCycle, 4);
-    }
-
-    /**
-     * As per the method above but for children turning 1.
-     *
-     * @param paymentCycle The current PaymentCycle including the relevant dates of birth of the children.
-     * @return The number of children who will soon turn 1 that affect the next Payment.
-     */
-    public int getNumberOfChildrenTurningOneAffectingNextPayment(PaymentCycle paymentCycle) {
-        return getNumberOfChildrenWithBirthdayAffectingNextPayment(paymentCycle, 1);
-    }
-
-    private int getNumberOfChildrenWithBirthdayAffectingNextPayment(PaymentCycle paymentCycle, int age) {
+    public NextPaymentCycleSummary getChildrenDateOfBirthSummaryAffectingNextPayment(PaymentCycle paymentCycle) {
```

Comment:
Suggest we call this `getNextPaymentCycleSummary`

Reply:
Yep, will do in the next PR.